python_modules/dagster-graphql/dagster_graphql/schema/instance.py

```diff
@@ -114,7 +114,9 @@ class GraphenePendingConcurrencyStep(graphene.ObjectType):
     stepKey = graphene.NonNull(graphene.String)
     enqueuedTimestamp = graphene.NonNull(graphene.Float)
     assignedTimestamp = graphene.Float()
-    priority = graphene.Int()
+    priority = (
+        graphene.BigInt()
+    )  # Maybe somebody will spam hold down the 9 key on their keyboard ¯\_(ツ)_/¯
```

Comment:
we actually enforce this in Python, see https://github.com/dagster-io/dagster/pull/25172/files